lib/pbio/include/pbio/os.h

```diff
@@ -176,6 +189,16 @@ struct _pbio_os_process_t {
         }                                       \
     } while (0)
 
+#define AWAIT_RACE(state, child1, child2, statement1, statement2)                    \
```

Comment:
I would include an explanation that we have to be really careful about the fact that there is no cancellation/cleanup of the thread that lost the race.